Weather/Models/ModelWeatherReport.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Weather.Models
+{
+    [Serializable]
```

Comment:
A Serializable attribútum csak olyan osztályokra kell, amiket Binárisan is akarsz szerializálni, régen ajánlás volt, hogy tedd ki, de mivel a Binary serializer már elavult és rajta kívül semmi nem eszi meg a Serializable attribútumot, így nem sokat ad hozzá.